api/HTMLScriptElement.json

```diff
@@ -330,6 +330,54 @@
           }
         }
       },
+      "integrity": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "45"
+            },
+            "chrome_android": {
+              "version_added": "45"
+            },
+            "edge": {
+              "version_added": "17",
+              "notes": "EdgeHTML did not pass all <a href='https://w3c-test.org/subresource-integrity/subresource-integrity.html'>W3C tests</a> for this property."
```

Comment:
I think this is a new note. Similar to in https://github.com/mdn/browser-compat-data/pull/7233, I don't think it's great to link to tests directly, when interpreting what the results mean takes a lot of knowledge.

https://wpt.fyi/results/subresource-integrity/subresource-integrity.html?product=edge-18 shows the results of the last Edge run we had of WPT. I'm not sure what the results mean in terms of what a web developer needs to know. I'd stick with `partial_implementation` unless it can be summarized what did and did not work.

Reply:
What do you recommend for a note, then?  If we're setting `partial_implementation`, we _must_ include a note explaining why (https://github.com/mdn/browser-compat-data/blob/master/docs/data-guidelines.md#partial_implementation-requires-a-note).

Reply:
I don't know what the test failures mean for a web developer, and that's really the problem. Without understanding what's missing from the implementation, setting `partial_implementation` isn't really possible.

If there's a way to avoid figuring this out in this PR that would be great. Otherwise, can you try to dig into the tests and understand what kind of code works differently in EdgeHTML compared to spec and other engines? It could be inconsequential stuff that isn't worth mentioning to web developers, or it could be that the feature doesn't work at all in common cases, I can't tell from a quick glance.

Reply:
I can't tell either, I don't know enough about this feature to really say.  Removing the note so we can get this in.